website/docs/starrocks.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 8
+title: "StarRocks"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Querying from StarRocks
+
+StarRocks allows you to query table formats like Hudi, Delta and Iceberg tables using our [external catalog](https://docs.starrocks.io/docs/data_source/catalog/catalog_overview/) feature.
+Users do not need additional configurations to work with OneTable synced tables.
+
+For more information and required configurations refer to:
+* [Hudi Catalog](https://docs.starrocks.io/docs/data_source/catalog/hudi_catalog/)
+* [Delta Lake Catalog](https://docs.starrocks.io/docs/data_source/catalog/deltalake_catalog/)
+* [Iceberg Catalog](https://docs.starrocks.io/docs/data_source/catalog/iceberg_catalog/)
+
+For hands on experimentation, please follow [Creating your first interoperable table](/docs/how-to#create-dataset)
+to create OneTable synced tables followed by [Hive Metastore](/docs/hms) to register the target table
+in Hive Metastore. Once done, please follow the below high level steps:
+1. Start the StarRocks server 
+2. From the directory where you have installed mysql-cli: login by running `mysql -P 9030 -h 127.0.0.1 -u root --prompt="StarRocks > "`.
+3. Add the external catalog (slight variation between open table formats)  `CREATE EXTERNAL CATALOG hudi_catalog_hms PROPERTIES ("type" = "hudi","hive.metastore.uris" = "thrift://hivemetastore:9083" );`.
```

Comment:
Can you put points 3 and 4 inside the `Tabs` block below aligning the examples for all three formats?